includes/class-sensei.php

```diff
@@ -588,7 +591,7 @@ public function update() {
 		}
 
 		// Mark site as having enrolment data from pre-3.0.0.
-		if ( version_compare( '3.0.0-dev', get_option( 'sensei-version' ), '>' ) ) {
+		if ( get_option( 'sensei-version' ) && version_compare( '3.0.0-dev', get_option( 'sensei-version' ), '>' ) ) {
```

Comment:
Just curious about what will happen if someone on Sensei pre-2.0 (where the version was named `woothemes-sensei-version`) updates to Sensei 3.0? There are still a fair number of people on such versions.

Reply:
This value will be set and Sensei should migrate. I assume they’d install SWCPC in that case so things should work.